src/Toml/Bi/Combinators.hs

```diff
@@ -298,3 +302,15 @@ list codec key = Codec
   where
     nonEmptyCodec :: TomlCodec (NonEmpty a)
     nonEmptyCodec = nonEmpty codec key
+
+set :: forall a . Ord a => TomlCodec a -> Key -> TomlCodec (Set a)
+set codec key = dimap (NE.fromList . S.toList) (S.fromList . NE.toList) nonEmptyCodec
+            where
+              nonEmptyCodec :: TomlCodec (NonEmpty a)
+              nonEmptyCodec = nonEmpty codec key
+
+hashSet :: forall a . (Hashable a, Eq a) => TomlCodec a -> Key -> TomlCodec (HashSet a)
```

Comment:
It would be also nice to add some documentation to these functions. At least in the spirit of the `list` function.

Reply:
Will go through the existing docs and add similar.